docs_en/tutorial/updating-our-app/index.md

```diff
@@ -9,7 +9,7 @@ Pretty simple, right? Let's make the change.
 
 ## Updating our Source Code
 
-1. In the `~/app/src/static/js/app.js` file, update line 56 to use the new empty text. ([Editing files in PWD tips here](/pwd-tips#editing-files))
+1. In the `~/app/src/static/app.js` file, update line 56 to use the new empty text. ([Editing files in PWD tips here](/pwd-tips#editing-files))
```

Comment:
The source is in `app/src/static/js`, but is a fairly recent change.

Reply:
Do you know when the change was made? Bcos the folks faced this on Saturday?

Reply:
Odd. It was made Friday my time, so let me run through it real quick and see if it's still an issue 👍

Reply:
Mk. This is what it looks like now. It's interesting that you have updated instructions, but not the updated zip, as those happened at the same time. Not sure what might be causing that.

<img width="235" alt="Screen Shot 2019-10-21 at 2 59 13 PM" src="https://user-images.githubusercontent.com/746837/67234569-63f3f880-f413-11e9-9b95-ba1589a6b52c.png">